src/software/embedded/constants/py_constants.py

```diff
@@ -0,0 +1,64 @@
+import os
+from enum import IntEnum
+import software.python_bindings as tbots_cpp
+
+DEFAULT_PRIMITIVE_DURATION = 2.0
+ROBOT_CONSTANTS = tbots_cpp.create2021RobotConstants()
+
+ROBOT_MAX_ANG_SPEED_RAD_PER_S = ROBOT_CONSTANTS.robot_max_ang_speed_rad_per_s
+ROBOT_MAX_SPEED_M_PER_S = ROBOT_CONSTANTS.robot_max_speed_m_per_s
+MAX_FORCE_DRIBBLER_SPEED_RPM = ROBOT_CONSTANTS.max_force_dribbler_speed_rpm
+
+
+class EstopMode(IntEnum):
+    """Enum for the various estop modes we can run thunderscope in
+
+    DISABLE_ESTOP: No physical / keyboard estop is needed, but we cannot send anything over the network
+    KEYBOARD_ESTOP: The spacebar can be used as an estop toggle instead of a physical estop
+    PHYSICAL_ESTOP: A physical estop is needed to run thunderscope, throws an exception if none is plugged in
+    """
+
+    DISABLE_ESTOP = 0
+    KEYBOARD_ESTOP = 1
+    PHYSICAL_ESTOP = 2
+
+
+# Paths to check for estop when running diagnostics
+ESTOP_PATH_1 = "/dev/ttyACM0"
+ESTOP_PATH_2 = "/dev/ttyUSB0"
+ESTOP_PATH_3 = "/dev/ttyUSB1"
```

Comment:
you can use udev rules (https://www.reactivated.net/writing_udev_rules.html#basic) to make the estop appear as `/dev/estop`